users/models.py

```diff
@@ -131,3 +131,17 @@ def save(self, **kwargs):
             self.last_vote_affirmation = timezone.now()
 
         return super().save(**kwargs)
+
+
+class UserGroup(models.Model):
+    name = models.CharField(max_length=255)
+    location = models.CharField(blank=True, null=True, max_length=255)
+    url = models.URLField('URL', blank=True, null=True)
+    url_type = models.IntegerField(choices=(
+        (0, 'Meetup'), 
```

Comment:
Can we turn this to the style we use in the `Job` model?

https://github.com/python/pythondotorg/blob/master/jobs/models.py#L112-L131

This way we could use these choices when we use the filter method. For example:

```py
UserGroup.objects.filter(url_type=UserGroup.TYPE_MEETUP)
```

Reply:
Sure, done.